### PR TITLE
Frontend: Footer Component #7

### LINKED ISSUE #7: Frontend: Footer Component
## User story

**Who:** **As a** developer
**What:** **I want to** have a Footer component where can show the links to the About us and Contact us Page, and link to the Create an Account
**Why:** **so that I can** have the same component in multiple pages.

## Acceptance criteria

Create a React component like the one we can see below
 
Screen Shot 2022-01-20 at 10 55 30

Check the wireframes here: https://www.figma.com/file/yo4QT2EcPo83HAMI3ydMxb/HYF-Wireframes?node-id=4%3A1324

## Implementation details

Download the images from the wireframes and add them to this folder src/client/assets/images

### PR DESCRIPTION
**Pull request by santhoshboinapally**
# Description
This PR is for the Footer React component .This Footer will displayed in all pages with same options to link these pages About us, Contact us , Creating new user account and email.

Fixes # 7

# How to test?
By running this command "npm run dev" you can test the file. 
Please provide a short summary how your changes can be tested?

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            <div className="footer-text">
+              <h4>COMPANY</h4>
+              <a href="/About">About us</a>
+              <br />
+              <a href="/Contact">Contact us</a>
+            </div>
+          </li>
+          <li className="footer-list">
+            <div className="footer-text">
+              <a href="/sign-up">
```
Comment (islam-fawzy25) [inline src/client/components/Footer/Footer.js]: I think it is better to use <Link to=""/> react router dom instead

**Comment by islam-fawzy25** [comment]
footerIpad
Footer in Ipad size looking good but if we align the logo to the left like other elements it will looks better 
and the same in mobile size "i can not see the logo and elements needs to be in column" 
footer

**Comment by islam-fawzy25** [comment]
Good Job @santhoshboinapally  💪🏻

File: src/client/components/Footer/Footer.js (hunk 1 above)
Comment (santhoshboinapally) [reply src/client/components/Footer/Footer.js]: Can i use Link 
Link to="/sign-up"<h4>CREATE` ACCOUNT</h4>/Link

**Comment by h09shais** [comment]
Hi @santhoshboinapally !
What is the reason to delete `src/client/containers/Home/Home.js` 😀 ?

**Comment by santhoshboinapally** [comment]
Hi @h09shais actually those files I have to commit with another branch, so I removed them and created another PullRequest.

File: src/client/containers/Home/Home.js [hunk 2]
```diff
@@ -1,3 +0,0 @@
-import React from 'react';
```
Comment (senner007) [inline src/client/containers/Home/Home.js]: Hi.

Why is the Home component removed here

File: src/client/components/Footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import send from '*../../../src/client/assets/images/emailsend.png';
+import './Footer.css';
+import { Link } from 'react-router-dom';
+
+function Footer() {
+  return (
+    <div>
+      <footer>
+        <ul>
+          <li className="footer-list">
```
Comment (senner007) [inline src/client/components/Footer/Footer.js:12]: At least the logo must be extracted into its own component to be reused elsewhere. Each of the links are separate components also

**Review by senner007 — CHANGES_REQUESTED** [review]

**Review by pankaj28843 — APPROVED** [review]
Looks good to me!

**Pull request merged**